src/metax_api/tests/api/rest/base/views/datasets/write.py

```diff
@@ -1016,7 +1137,6 @@ def test_catalog_record_partial_update_list_error_key_not_found(self):
         self.assertEqual('identifying key' in response.data['failed'][0]['errors']['detail'][0], True,
                          response.data['failed'][0]['errors'])
 
-
```

Comment:
I think there is commonly two empty lines between class definitions (for no other reason that easier for eyes to parse code).